go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/olekukonko/tablewriter v0.0.4
 	github.com/ory/dockertest/v3 v3.6.2
 	github.com/oschwald/geoip2-golang v1.4.0
+	github.com/otiai10/copy v1.3.0
```

Comment:
Used for copying folders (for tests).
In some other migration tests I have some custom code for copying folders that we can change in the future to simply use this lib too.